gravitee-am-test/README.adoc

```diff
@@ -0,0 +1,46 @@
+= Integration Tests
+
+The AM integration tests run on Jest using Supertest to make calls to the AM APIs.
+To run the tests from a terminal you need to be in the `gravitee-am-test` directory.
+
+==== Build
+
+Execute `npm install`. You usually only need to do this once, i.e. after cloning the AM repo or when dependencies have changed.
+
+==== Run tests with default config
+
+Executing `npm test` or `jest` will run all the tests. To run individual tests use `npm run` | `jest -t`, followed by the name of the spec file.
```

Comment:
instead of `jest` perhaps the use has to run 'jest --config=api/config/dev.config.js' after executing 'npm install jest'.